serverless/sql-databases/quickstart.mdx

```diff
@@ -0,0 +1,95 @@
+---
+meta:
+  title: SQL Databases - Quickstart
+  description: This page shows you how to get started with Scaleway Serverless SQL Databases
+content:
+  h1: SQL Databases - Quickstart
+  paragraph: This page shows you how to get started with Scaleway Serverless SQL Databases
+tags: sql-databases serverless database namespace connect create delete
+dates:
+  validation: 2023-07-17
+  posted: 2023-07-17
+categories:
+  - serverless
+  - sql-databases
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+  - You have installed [PostgreSQL](https://www.postgresql.org/download/).
+</Message>
+
+Scaleway Serverless SQL Databases are fully managed databases that automatically scale both in storage and compute according to your needs.
+
+This page explains how to create, access and delete a Serverless SQL Database using the Scaleway Console.
+
+## How to create a database
+
+1. Click **SQL Databases** in the **Serverless** section of the side menu. The Serverless SQL Databases page displays.
+
+2. Click **+ Create database**. The database creation wizard displays.
+
+3. Complete the following steps in the wizard:
+
+    - Choose a **region**, which is the geographical location in which your database will be created.
+
+    - Select a **default engine version** from the drop-down menu.
+
+    - Configure your **database autoscaling**, or keep the default values. Your database will scale within the defined range to match the incoming load.
+
+    <Message type="note">
+        Define a minimum of 1 vCPU or more to avoid cold starts that can take a few seconds. 
+    </Message>
+
+    - Enter a **name** for your database, or keep the randomly-generated name. The name must only contain alphanumeric characters and dashes.
+
+    - Verify the **estimated cost**.
+
+4. Click **Create database** to finish. The setting up of your database will take a few minutes. Check the **Overview** tab to see its status.
+
+## How to connect to a database
+
+1. From the **Overview** tab of your database, click the **Set up credentials** button. The wizard displays.
```

Comment:
```suggestion
1. From the **Overview** tab of your database, click the **Set up credentials** button. The creation wizard displays.
```

Reply:
It's not a creation, but a setup, so I'll write "setup wizard"